tasks/sep/fp-recovery/003-enable-permissionless-game/README.md

```diff
@@ -0,0 +1,54 @@
+# Deputy Guardian - Enable Permissioness Dispute Game
+
+This task updates the `respectedGameType` in the `OptimismPortal` to `CANNON`, enabling users to permissionlessly propose outputs as well as for anyone to participate in the dispute of these proposals. This action requires all in-progress withdrawals to be re-proven against a new `FaultDisputeGame` that was created after this update occurs.
+
+The batch will be executed on chain ID `11155111`, and contains `1` transactions.
+
+## Tx #1: Update `respectedGameType` in the `OptimismPortal`
+
+Updates the `respectedGameType` to `CANNON` in the `OptimismPortal`, enabling permissionless proposals and challenging.
+
+**Function Signature:** `setRespectedGameType(address,uint32)`
+
+**To:** `0x4220C5deD9dC2C8a8366e684B098094790C72d3c`
+
+**Value:** `0 WEI`
+
+**Raw Input Data:** `0xa1155ed900000000000000000000000016fc5058f25648194471939df75cf27a2fdc48bc0000000000000000000000000000000000000000000000000000000000000000`
+
+### Inputs
+
+**\_gameType:** `0`
+
+**\_portal:** `0x16Fc5058F25648194471939df75CF27A2fdC48BC`
+
+## Preparing the Operation
+
+1. Collect signatures for the action.
+
+1. Execute the action.
+
+## Signing and execution
+
+Please see the signing and execution instructions in [SINGLE.md](../../../../SINGLE.md).
+
+### State Validations
+
+The only state modification that is made by this action is an update to the shared slot between the `respectedGameType`
+and `respectedGameTypeUpdatedAt` variables:
+
+![state-diff](./images/state_diff.png)
+
+Slot `0x000000000000000000000000000000000000000000000000000000000000003b` in the `OptimismPortal` proxy has the following packed layout:
```

Comment:
In each of the 3 readme files, please link to the location in the storage layout lockfiles which will attest to the relevant slots.